ocs_ci/ocs/ui/base_ui.py

```diff
@@ -116,11 +116,11 @@ def __init__(self):
         )
         if not os.path.isdir(self.screenshots_folder):
             Path(self.screenshots_folder).mkdir(parents=True, exist_ok=True)
-        logger.info(f"screenshots folder:{self.screenshots_folder}")
+        logger.debug(f"screenshots folder:{self.screenshots_folder}")
```

Comment:
I think it is better to use `info` level and then we can see the screenshots location when we run the test via pycharm...

Reply:
discussed